addons/rook/1.4.3/install.sh

```diff
@@ -13,7 +13,7 @@ function rook_pre_init() {
             SKIP_KUBERNETES_HOST=1
         fi
     elif [ "$ROOK_BLOCK_STORAGE_ENABLED" != "1" ]; then
-        bail "Rook 1.4.3 requires enabling block storage"
+        logWarn "Rook 1.4.3 requires enabling block storage"
```

Comment:
What's the point to warning here?

Why not just remove all references to `ROOK_BLOCK_STORAGE_ENABLED` in 1.4.3+?

Reply:
Well the goal was to inform users that the configuration was going to be ignored. Really I just wanted something temporary so that I could launch in test grid and it wouldn't fail. Obviously it's failing to create TG runs while that api validation is throwing errors for Rook version > 1.4.* without an explicit isBlockStorageEnabled.

Reply:
On further thought this warning will probably only confuse people.